persistence-utils-demo-webtest/src_test/com/axonivy/utils/persistence/webtest/WebTestDemoIT.java

```diff
@@ -140,7 +144,7 @@ public void testEditPerson() {
 		$(By.id("personForm:maritalStatus")).click();
 
 		$(By.id("personForm:birthdate_input")).shouldBe(enabled).clear();
-		$(By.id("personForm:birthdate_input")).shouldBe(enabled).sendKeys("02.12.1982");
+		$(By.id("personForm:birthdate_input")).shouldBe(enabled).sendKeys("02.12.1982", Keys.TAB);

```

Comment:
very common problem: I think we should extend our PrimeUi helpers with a 'datePicker' helper. @ivy-lli